lib/execution_engine2/sdk/EE2Runjob.py

```diff
@@ -46,6 +46,10 @@
     DEBUG_MODE,
 )
 from execution_engine2.utils.job_requirements_resolver import RequirementsType
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from lib.execution_engine2.sdk.SDKMethodRunner import SDKMethodRunner
```

Comment:
Codacy is complaining about this being an unused import, not sure if there's a trivial way to get it to ignore it, if not don't worry about it

Reply:
yea it's impossible to get rid of it from codacy from what Ive researched

Reply:
:+1: